cilib/service/charm.py

```diff
@@ -30,13 +30,15 @@ def sync(self):
             return
 
         self.log(f"Syncing {self.upstream_normalized} -> {self.downstream_normalized}")
+        upstream_ref = self.repo.default_gh_branch("upstream")
+        downstream_ref = self.repo.default_gh_branch("downstream")
```

Comment:
```suggestion
        upstream_ref = self.repo.default_gh_branch(self.upstream_normalized)
        downstream_ref = self.repo.default_gh_branch(self.downstream_normalized)
```

These need to be repo url strings, right?  Or perhaps your intention was to make `repo.default_gh_branch` handle the literal "upstream" vs "downstream" as a param?